search_that_hash/cracker/online_mod/online.py

```diff
@@ -207,7 +207,7 @@ def crack(hashes):
         url = "https://av5b81zg3k.execute-api.us-east-2.amazonaws.com/prod/lookup"
         payload = json.dumps({"Hash": hashes})
         headers = {"Content-Type": "application/json"}
-        response = requests.request("GET", url, headers=headers, data=payload)
+        response = requests.request("GET", url, headers=headers, data=payload, timeout=1)

```

Comment:
Shouldn't this timeout be associated with the argument `--timeout`?

Reply:
Fixed

Reply:
No you didn't :)

Reply:
Every other timeout in that file is `hashes[4]`, I also think it'd be easier if you passed in kwargs instead and used that to inform the decisions of what the searcher does as this allows us to expand in the future :D 

TL;DR:

Pass Kwargs into the constructor 
```python
def __init__(self, kwargs)
    self.args = kwargs
```

Reply:
I put config and not kwargs as config is basically the more "updated" version of kwargs :)

Reply:
I will do config instead of kwargs because thats more updated and is already in there :)